File: Örnek-6.15/C++/devre.cpp
#include <iostream>
using namespace std;
float gerilimK ,R1,R2,R3,R4,akim,vr1,pr2;

int main() {
	cout<<"gerilim degeini giriniz:";
	cin>>gerilimK;
	cout<<"R1 R2 R3 R4 (ohm) cinsinden giriniz: ";
	cin>>R1>>R2>>R3>>R4;
	
	akim=gerilimK/(R1+R2+R3+R4);
	vr1=R1*akim;
	pr2=akim*akim*R2;
	
	cout<<"\ndevreden gecen akim: "<<akim<<"amper";
	cout<<"\nR1 direncine dusen gerilim:"<<vr1<<"volt";
	cout<<"\nR2 direncinde harcanan guc:"<<pr2<<"watt";
	return 0;
}

File: Örnek-6.15/Java/devre.java
package algoritma;
import java.util.Scanner;

public class devre {

	
	public static void main(String[] args) {
		float akim,r1,r2,r3,r4,gerilimK,vr1,pr2;
		Scanner scan=new Scanner(System.in);
		
		System.out.print("gerilim kaynağının değerrini giriniz (volt):");
		gerilimK=scan.nextFloat();
		
		System.out.print("R1 R2 R3 R4 değerlerini (ohm) cinsinden giriniz:");
		r1=scan.nextFloat();
		r2=scan.nextFloat();
		r3=scan.nextFloat();
		r4=scan.nextFloat();
		
		akim=gerilimK/(r1+r2+r3+r4);
		vr1=r1*akim;
		pr2=akim*akim*r2;
		
		System.out.println("deveden akan akim: "+akim+" amper");
		System.out.println("R1 direncine düşen gerilim: "+vr1+" volt");
		System.out.println("R2 direncinde harcanan güç: "+pr2+" watt");
		
		
	}

}
